src/index.js

```diff
@@ -2,8 +2,21 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import { Provider } from 'react-redux';
-import App from './App';
-import store from './app/store';
+import { createStore } from 'redux';
+import reducer from './reducers';
+import App from './Components/App';
+
+const initialState = [
+  { id: Math.floor(Math.random() * 100) + 1, title: 'Book1', category: 'Horror' },
+  { id: Math.floor(Math.random() * 100) + 1, title: 'Book2', category: 'Action' },
+  { id: Math.floor(Math.random() * 100) + 1, title: 'Book3', category: 'Biography' },
+  { id: Math.floor(Math.random() * 100) + 1, title: 'Book4', category: 'History' },
+  { id: Math.floor(Math.random() * 100) + 1, title: 'Book5', category: 'Kids' },
+  { id: Math.floor(Math.random() * 100) + 1, title: 'Book6', category: 'Learning' },
```

Comment:
### [Optional]
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)

Example:
```
import uuid from 'uuidv4'
id: uuid(): {title: 'Book1', category: 'Horror'}
```